CHANGELOG.md

```diff
@@ -18,6 +18,16 @@
  - Fix virus count sorting on "Virus Report"
  - Remove additional slashes in "SpamAssassin Rule Description Update"
  - Adding translation to user interface
+ - Add LOGO path in conf.php.example and changes done in corresponding files.
+ - layout changes in quarantine.php, other.php, sf_version.php
+ - Removed duplicate PHP function in tools/Cron_jobs/quarantine_report.php (is in functions.php)
+ - Changes in tools/Sendmail_relay/INSTALL
+ - Changes in tools/Sendmail_relay/sendmail_relay.init
```

Comment:
I removed rc.d in  /etc/rc.d/init.d/ and .init

Uptodate for Debian and Ubuntu.

The last version for tools/Sendmail_relay/INSTALL:

# cp tools/Sendmail_relay/sendmail_relay.php /usr/local/bin/.
# cp tools/Sendmail_relay/sendmail_relay.init /etc/init.d/.
# chmod +x /etc/init.d/sendmail_relay
# /etc/init.d/sendmail_relay.init start
# update-rc.d sendmail_relay.init defaults